### DISCUSSION TITLE
Creating a Blog With GitHub Pages and Hugo

### DISCUSSION THREAD

**Discussion by giscus** [elibroftw/blog.elijahlopez.ca#16] [General]
# Creating a Blog With GitHub Pages and Hugo

I created this blog using Hugo and host it on GitHub Pages. In this tutorial, I’ll teach you to do the same.
The resources I had used to create a working blog left out some important details that I will include in my tutorial.
I’ve also done the brunt of the work to perfect the theme and workflow.
If you are like me, you can follow the tutorial that forks my site rather than making a blog site from scratch.

https://blog.elijahlopez.ca/posts/hugo-tutorial/

**Comment by elibroftw** [comment]
Test comment section.

**Comment by konrad-wila** [comment]
This post seems outdated the website deploys correctly using the default Hugo GitHub action instead of deploy from branch and the comments section is using giscus instead of Utterances.
